Examples/Docs/Sources/AsyncExamples/main.swift

```diff
@@ -18,7 +18,7 @@ private func causalConsistency() throws {
     let s1 = client1.startSession(options: ClientSessionOptions(causalConsistency: true))
     let currentDate = Date()
     var dbOptions = DatabaseOptions(
-        readConcern: ReadConcern(.majority),
+        readConcern: ReadConcern.majority,
```

Comment:
I thought the idea was that we could write like:
```
let dbOptions = DatabaseOptions(
    readConcern: .majority,
    ...
)
```

Reply:
Indeed that was the point! I gotta level up my swifiness 🏃 🏃 🏃